src/components/modal/CdrModal.vue

```diff
@@ -190,6 +201,62 @@ const removeNoScroll = () => {
   }
 };
 
+const backgroundContentNodes: Array<HTMLElement> = [];
+let backgroundNodesDiscovered = false;
+
+/**
+ * findBackgroundContentNodes
+ * When modal opens for the first time, find the content that should
+ * be hidden when it opens, and store it in an array
+ */
+const findBackgroundContentNodes = () => {
+  // initial selector is complex, breaking it down
+  const notSelectors = [
+    'body > *',
+    ':not(script)',
+    ':not(style)',
+    ':not([data-is-modal])',
+    ':not([aria-hidden=true])',
+  ]
```

Comment:
Summary of these selectors: find all direct children of the body element, that are not:

- script or style tags
- that are not this modal or other modals (they should be hidden when the top modal is open anyway)
- that are not already `aria-hidden`